CMakeLists.txt

```diff
@@ -143,14 +147,16 @@ if (ALUMINUM_ENABLE_BENCHMARKS)
 endif ()
 
 # Setup CXX requirements
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic -g")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic -g3")
 # -faligned-new is needed to use new alignment-aware new when available.
 include(CheckCXXCompilerFlag)
 check_cxx_compiler_flag("-faligned-new" CXX_COMPILER_HAS_FALIGNED_NEW)
 if (CXX_COMPILER_HAS_FALIGNED_NEW)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -faligned-new")
 endif ()
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+set(CMAKE_CXX_FLAGS_DEBUG "-Og")
```

Comment:
This option is not portable. Also, if you're going to clobber the whole variable, you probably want `"-Og -g"`.